### PR TITLE
Remove News link from site

### PR DESCRIPTION
**Pull request by lonnieezell**
We don't really have the hands needed to keep an active news section going, so I'm ripping out the links to the news links and routes. I'm leaving the blog engine itself, so that it's easy to add back in if we ever decide to.

### CODE REVIEW

**Comment by kenjis** [comment]
I don't know how much sense it makes to remove the link.

If there is no link, nobody can reach the past articles.

**Comment by datamweb** [comment]
I was past this PR but I was really upset.
I think the team should try to introduce the product more, not to eliminate the insufficient amount.

With all due respect, I am 100% against this PR.

I know that you have put a lot of effort and sacrificed free time, but this is an important part of the official website that can announce the results of the team.

**Comment by demyr17** [comment]
Hi Lonnie, 
I do not know what you mean by "active" but having a _**news**_ section is important for any project, even with rare posts. And we publish something. Our latest news is not from a year ago! Therefore, unless you have a new idea to replace the current news section, I will object this idea.

**Comment by lonnieezell** [comment]
I would agree that having a well-maintained news section is a good feature. I would submit, though, that having a news section that is almost exclusively used for release notifications, but has missed the last 7 releases has a negative impact and makes the project look abandoned or dying. 

We already have links to most recent announcements section of the forum on the front page of the website, so any posts about releases are often duplicates or poor substitutes. My point is that we're not losing any information - it is still there on the site - but we're having a better appearance to newcomers. 

The only other solution I see is to form a team that regularly puts out content with information about new projects built with it from our users, tutorials, etc. And I don't see anyone stepping up to do that. Since everything is open-source and doesn't have a big commercial entity behind it like a lot of other frameworks, it makes it more challenging to keep a PR team together, since we're all just doing this when we have time and energy to volunteer.

File: public/assets/css/ci-theme.css [hunk 1]
```diff
@@ -6,271 +6,313 @@
...
+            format("embedded-opentype"),
+        /* IE6-IE8 */ url("../fonts/raleway-v14-latin-200.woff2")
+            format("woff2"),
+        /* Super Modern Browsers */ url("../fonts/raleway-v14-latin-200.woff")
+            format("woff"),
+        /* Modern Browsers */ url("../fonts/raleway-v14-latin-200.ttf")
+            format("truetype"),
+        /* Safari, Android, iOS */
+            url("../fonts/raleway-v14-latin-200.svg#Raleway") format("svg"); /* Legacy iOS */
+}
```
Comment (kenjis) [inline public/assets/css/ci-theme.css:27]: This commit message is "Remove News link from site".
Can you remove the unrelated changes in this commit?
Because it makes difficult to review.

Do the changes in this file change the CSS code, or just fixing coding style?

**Comment by kenjis** [comment]
> We already have links to most recent announcements section of the forum on the front page of the website, so any posts about releases are often duplicates or poor substitutes. My point is that we're not losing any information - it is still there on the site - but we're having a better appearance to newcomers.

What do you mean?

First, this PR does not change the "CodeIgniter Forum" part. The links to news are still there:
Screenshot 2023-08-17 10 02 57

File: public/assets/css/ci-theme.css (hunk 1 above)
Comment (lonnieezell) [reply public/assets/css/ci-theme.css:27]: Well damn. Didn't realize my editor reformatted everything. I made a small change to the width of the menu only so that the removal of a link didn't leave a gap between the remaining links and the "Download" button.

**Comment by lonnieezell** [comment]
> > We already have links to most recent announcements section of the forum on the front page of the website, so any posts about releases are often duplicates or poor substitutes. My point is that we're not losing any information - it is still there on the site - but we're having a better appearance to newcomers.
> 
> What do you mean?
> 
> First, this PR does not change the "CodeIgniter Forum" part. The links to news are still there: 

That's my point. The links to the forum posts are where we post the release information. All the blog was really being used for was telling about the releases. And it was a very long way behind (the last post was 8 months ago about 4.3 release, and we're up to 4.3.7 now). The blog was duplicating what is already available in the forum links. 

Since we don't have the hands to do more with it, I feel we need to remove it because right now it's providing a tarnished image of the project. Having no blog at all, and just using the forums like we've done in the past, is enough to keep everything current and make it look like the project is still active.

**Comment by kenjis** [comment]
@lonnieezell The links are links to the news articles in the site. If we don't update News, the links never change.
Perhaps, you mean to output links to the forum news posts (https://forum.codeigniter.com/forumdisplay.php?fid=2)?
If so, we need to create a ViewCell for it.

**Comment by lonnieezell** [comment]
Doh! Silly me. Pretty sure that used to be pulling links from the announcement forum. Forgot I changed it over to the blog instead. Well, then, if people are ok with it, I'll make that happen and replace that with posts from the forums.

**Comment by kenjis** [comment]
> I feel we need to remove it because right now it's providing a tarnished image of the project. 

I understand your concern, so I'm okay with the change.

**Comment by demyr17** [comment]
Hey. What about replacing “news” with “forum” in the top menu? It will require no change in the css and the forum is really important. We have to scroll down to visit the forum.  (To be honest, in terms of UX we have had this problem for a long time. It should be easy to catch.)

**Comment by MGatner** [comment]
Late to this convo, but I want to throw out one more controversial idea 🙃

LLMs are very good at generating summaries of recent activity for blog posts - what if we used that to keep something relevant going in the News section until we have a minor release that is worth a human write-up?

**Comment by MGatner** [comment]
> Write a new blog post entry as if you are a member of the CodeIgniter development team. Note that there are no big announcements right now but hint at some exciting upcoming features. Note that there are some new contributing members and the project continues to grow. Don’t make it long, and keep the tone informal.

Hello CodeIgniter Community,

Greetings from the CodeIgniter development squad! We're here to give you a sneak peek into our world, where exciting things are brewing – without the big drumroll just yet.

Let's start by giving a warm shout-out to the new members who've hopped on board. The team's growing, and their fresh ideas are injecting some serious energy into the mix. Keep your eyes peeled for the cool stuff they're cooking up!

As for what's on our horizon, we're cranking away on some nifty features. Think smoother-than-ever interactions with RESTful APIs – we're leveling up your backend game. And that's not all – we've got performance boosts and security enhancements in our sights too. Our goal? To keep CodeIgniter your trusty partner in all things web development.

So, while we might not have fireworks to show off today, know that the wheels are turning. Thanks for being part of our journey, and stay tuned for the updates that'll make your coding heart skip a beat.

Catch you soon,
[Your Name]
From the CodeIgniter Dev Team

**Comment by kenjis** [comment]
> Write a new blog post entry as if you are a member of the CodeIgniter4 development team. Note that there are no big announcements right now but hint at some exciting upcoming features. Note that there are some new contributing members and the project continues to grow. Don’t make it long, and keep the tone informal.

Hey CodeIgniter enthusiasts!

Hope you're all doing great and enjoying your coding adventures with CodeIgniter 4! We've been buzzing away behind the scenes, and even though we don't have any major announcements just yet, we couldn't wait to share some of the cool stuff we've got brewing.

First things first, we want to give a warm welcome to our new contributing members who have hopped onto the CodeIgniter train. 🎉 The family is growing, and with fresh minds come fresh ideas. These folks have been diving into the codebase, submitting pull requests, and sharing their insights. It's always amazing to see the community spirit alive and kicking!

Now, about those exciting upcoming features. We don't want to spill all the beans just yet, but let's just say we're tinkering away on a couple of things that might just make your coding life even smoother. Expect some enhancements to our validation system that'll make form handling a breeze, and keep an eye out for a revamp of our database interactions that'll leave you with more time for your favorite cup of coffee.

Oh, and if you've been itching for some performance boosts, stay tuned. We're squeezing out every last drop of speed we can, so your applications will be blazing fast.

As always, we're here because of you and for you. Your feedback, suggestions, and bug reports are the fuel that keeps this project running. So don't hesitate to hop onto our forums, chat rooms, or GitHub to let us know what's on your mind.

That's all for now, folks! Keep coding, keep exploring, and keep an eye out for those updates just around the corner. Until next time, happy coding!

Cheers,
[Your Name]
CodeIgniter4 Dev Team

**Comment by lonnieezell** [comment]
Those generated posts can be generated and posted in tne forum and work just as well. 

I pushed changes that replaced the home page Recent News section to pull from the forum instead.

**Comment by MGatner** [comment]
I will pull locally and give it a look.

**Comment by demyr17** [comment]
> Hey. What about replacing “news” with “forum” in the top menu? It will require no change in the css and the forum is really important. We have to scroll down to visit the forum. (To be honest, in terms of UX we have had this problem for a long time. It should be easy to catch.)

@lonnieezell maybe you have missed it? What about this suggestion?

File: app/Database/Migrations/2020-10-29-035422_CreateForumsTable.php [hunk 2]
```diff
@@ -20,6 +20,7 @@ public function up()
     {
         // fx_threads
         $this->forge->addField([
+            'fid'        => ['type' => 'int', 'constraint' => 11, 'unsigned' => true, 'autoincrement' => true],
```
Comment (kenjis) [inline app/Database/Migrations/2020-10-29-035422_CreateForumsTable.php:23]: @lonnieezell How do we deploy this update?

**Comment by kenjis** [comment]
I cannot see the difference in the CSS file.
I checked the screen, and it seems okay.

Firefox:
Screenshot 2023-08-31 14 12 04
Screenshot 2023-08-31 14 12 18

Chrome:
Screenshot 2023-08-31 14 13 20

**Comment by lonnieezell** [comment]
> > Hey. What about replacing “news” with “forum” in the top menu? It will require no change in the css and the forum is really important. We have to scroll down to visit the forum. (To be honest, in terms of UX we have had this problem for a long time. It should be easy to catch.)
> 
> @lonnieezell maybe you have missed it? What about this suggestion?

While I'm not totally opposed to that, I think that's up for discussion. That ignores Slack and is a semi-duplicate of the `Discuss` link already there. Mid-way down the page we have a direct link to the forums, also.

File: app/Database/Migrations/2020-10-29-035422_CreateForumsTable.php (hunk 2 above)
Comment (lonnieezell) [reply app/Database/Migrations/2020-10-29-035422_CreateForumsTable.php:23]: We don't need to. This is solely for local installations that need to simulate the MyBB forums.

**Comment by lonnieezell** [comment]
> I cannot see the difference in the CSS file. I checked the screen, and it seems okay.
> 

Then if we're ok with it all someone just needs to approve it :)

File: app/Database/Migrations/2020-10-29-035422_CreateForumsTable.php (hunk 2 above)
Comment (kenjis) [reply app/Database/Migrations/2020-10-29-035422_CreateForumsTable.php:23]: Okay, I got it.

**Review by kenjis — APPROVED** [review]

**Comment by kenjis** [comment]
@demyr17 @lonnieezell Replacing “news” with “forum” in the top menu is pending.
Create a new PR if necessary.
It is better to merge this PR first, so merge it.
